TcxTools.sln

```diff
@@ -1,29 +1,25 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
+VisualStudioVersion = 15.0.28010.2019
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TcxTools", "TcxTools\TcxTools.csproj", "{EC8171C5-D86C-4F08-A080-BC61DE101E52}"
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{8B11D68E-E33C-4C51-A94A-2BD68E2AF7C3}"
```

Comment:
Why remove these Solution Items?

Reply:
In VS 2017, nuget is supported natively by VS. The separate nuspec is no longer needed. I carried over the information, please check the added properties inside TcxTools.csproj. 
For details, refer here: [create-and-publish-a-package-using-visual-studio](https://docs.microsoft.com/en-us/nuget/quickstart/create-and-publish-a-package-using-visual-studio)